Makefile

```diff
@@ -15,7 +15,7 @@
 
 PKG=github.com/kubernetes-sigs/aws-efs-csi-driver
 IMAGE?=amazon/aws-efs-csi-driver
-VERSION=v1.2.1-dirty
+VERSION=v1.3.0-dirty
```

Comment:
we can solve it later but seems weird how we always append dirty? i understand 'dirty' to mean it's a build from a git repo that has uncommitted changes

Reply:
Should we get rid of `dirty`?

Reply:
mmmmmmm I guess that's a quick fix. doesn't seem valuable.